browser-test/src/admin/admin_program_creation.test.ts

```diff
@@ -16,6 +16,7 @@ test.describe('program creation', () => {
 
   test('create program page', async () => {
     const {page, adminPrograms, adminProgramImage} = ctx
+    await enableFeatureFlag(page, 'disabled_visibility_condition_enabled')
```

Comment:
suggestion (blocking): my preference would be to add a new test that tests functionality/UI when the flag is on instead of enabling the feature flags in the existing tests.